projects/train/train/models/flow.py

```diff
@@ -161,9 +162,25 @@ def test_step(self, batch, _):
         self.idx += 1
 
     def on_test_epoch_end(self):
+        # pp plot
         bilby.result.make_pp_plot(
             self.test_results,
             save=True,
             filename=self.outdir / "pp-plot.png",
             keys=self.inference_params,
         )
+
+        # searched area cum hist
+        searched_areas = []
+        for result in self.test_results:
+            searched_area = result.calculate_searched_area()
+            searched_areas.append(searched_area)
+        searched_areas = np.sort(searched_areas)
+        counts = np.arange(1, len(searched_areas) + 1) / len(searched_areas)
+
+        plt.figure(figsize=(10, 6))
+        plt.step(searched_areas, counts, where="post")
+        plt.xlabel("Searched Area (deg^2)")
```

Comment:
Generally, a log scaling on the x-axis is able to show the plot better, but it may not always work. Shall we plot two versions of the plot? One with the log scale and one with linear scale?

Reply:
I think you're right. I actually haven't fully tested this , working on it now. Let me do that and I'll post the plot in here

Reply:
Okay got it going - Yes xaxis log scale looks much better just pushed